sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,19 +1,27 @@
 # Release History
-## 7.0.0-beta.6 (Unreleased)
-- Exposing Amqp Message envelope in form of `AmqpAnnotatedMessage` as a property of `ServiceBusReceivedMessage` and `ServiceBusMessage`.
-- Removed `ServiceBusReceiverClientBuilder.maxAutoLockRenewalDuration`. Use method `getAutoRenewMessageLock` of classes `ServiceBusReceiverClient`
-  and `ServiceBusReceiverAsyncClient` to lock messages and sessions.
-- Updated datetime related APIs to use `java.time.OffsetDateTime` instead of `java.time.Instant`.
-- Removed `scheduledMessageCount` from `SubscriptionRuntimeInfo` and added it to `TopicRuntimeInfo`.
-- Changed `QueueRuntimeInfo`, `TopicRuntimeInfo` and `SubscriptionRuntimeInfo` to `QueueRuntimeProperties`, `TopicRuntimeProperties`
-  and `SubscriptionRuntimeProperties` respectively.
- 
+
+## 7.0.0-beta.6 (2020-09-11)
```

Comment:
We also have this unreleased dependency for service-bus [here](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L132) 
Do we still use it?